package-lock.json

```diff
@@ -2705,20 +2716,17 @@
       "from": "github:luigiplr/node-openvpn",
       "dev": true,
       "requires": {
-        "bluebird": "^3.5.2",
-        "lodash": "^4.17.11",
-        "moment": "^2.22.2"
+        "bluebird": "^2.9.34",
```

Comment:
@shumkov @antouhou  Any idea why this is _downgrading_ versions of these modules? Is it possible this bot is not yet prod ready? Might be intentional, but wanted to check.

Reply:
Hm... No Idea. I would try `npm audit fix` and see the result. I feel like this bot uses the same command internally.